packages/web-components/src/text/text.stories.ts

```diff
@@ -0,0 +1,119 @@
+import { html } from '@microsoft/fast-element';
+import type { Args, Meta } from '@storybook/html';
+import { renderComponent } from '../__test__/helpers.js';
+import type { Text as FluentText } from './text.js';
+import './define.js';
+
+type TextStoryArgs = Args & FluentText;
+type TextStoryMeta = Meta<TextStoryArgs>;
+
+/**
+ * Used to generate slotted content for stories
+ * @param as - the element to generate
+ * @param content - the content for the element
+ * @returns ViewTemplate
+ */
+const generateSemanticElementTemplate = (as: string, content) => {
+  switch (as) {
+    case 'h1':
+      return html`<h1>${content}</h1>`;
+    case 'h2':
+      return html`<h2>${content}</h2>`;
+    case 'h3':
+      return html`<h3>${content}</h3>`;
+    case 'h4':
+      return html`<h4>${content}</h4>`;
+    case 'h5':
+      return html`<h5>${content}</h5>`;
+    case 'h6':
+      return html`<h6>${content}</h6>`;
+    case 'p':
+      return html`<p>${content}</p>`;
+    case 'pre':
+      return html`<pre>${content}</pre>`;
+    case 'span':
+    default:
+      return html`<span>${content}</span>`;
+  }
+};
+
+const storyTemplate = html<TextStoryArgs>`
+  <fluent-text
+    align=${x => x.align}
+    font=${x => x.font}
+    size=${x => x.size}
+    weight=${x => x.weight}
+    ?nowrap=${x => x.nowrap}
+    ?truncate=${x => x.truncate}
+    ?italic=${x => x.italic}
+    ?underline=${x => x.underline}
+    ?strikethrough=${x => x.strikethrough}
+    ?block=${x => x.block}
+    >${x => generateSemanticElementTemplate(x.as, x.content)}</fluent-text
+  >
+`;
+
+export default {
+  title: 'Components/Text',
+  args: {
+    content: 'Text',
+    nowrap: false,
+    truncate: false,
+    italic: false,
+    underline: false,
+    strikethrough: false,
+    block: false,
+  },
+  argTypes: {
+    as: {
+      options: ['h1', 'h2', 'h3', 'h4', 'h5', 'h6', 'p', 'pre', 'span'],
+      control: {
+        type: 'select',
+      },
+    },
+    size: {
+      options: ['100', '200', '300', '400', '500', '600', '700', '800', '900', '1000'],
```

Comment:
If we stick to object + type in options file, we should use `Object.keys(TextSize)` to avoid code duplication.

Reply:
Good catch - this was copy/paste from elsewhere :)